plugins/guests/redhat/cap/configure_networks.rb

```diff
@@ -43,13 +43,18 @@ def self.configure_networks(machine, networks)
               /sbin/ifdown '#{network[:device]}' || true
 
               # Move new config into place
-              mv '#{remote_path}' '#{final_path}'
+              mv -f '#{remote_path}' '#{final_path}'
 
               # Bring the interface up
               ARPCHECK=no /sbin/ifup '#{network[:device]}'
             EOH
           end
 
+          commands << <<-EOH.gsub(/^ {14}/, '')
+            # Restart network
+            service network restart
+          EOH
+
```

Comment:
Can you explain what the network restart is providing here? Just want to confirm that I understand the reasoning for the addition. Thanks!